docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -605,6 +610,46 @@ export default async function buildComponentsDocumentation(
   if (fails.length > 0) {
     process.exit(1);
   }
+
+  // Build charts API page indexes
```

Comment:
Can we extract that into a standalone method?
Would probably be easier to read.